CONTRIBUTING.md

```diff
@@ -30,9 +30,9 @@ feature branches off of `master`, and their pull requests should target the
 requests.
 
 Pull requests against `master` trigger automated tests that are run through
-Azure DevOps. Additional test suites are run periodically. When adding new code
-paths or features, tests are a requirement to complete a pull request. They
-should be added in the `test` directory.
+Azure DevOps or GitHub Actions. Additional test suites are run periodically.
```

Comment:
I know we might move to GitHub Actions in the future, but this does seem a little premature.

Reply:
How so? It's part of the existing automation.